src/tests.builds

```diff
@@ -3,12 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <ItemGroup>
-    <TestProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\System.Security.AccessControl.Tests.csproj" />
     <TestProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\WebServer.csproj" Condition="'$(OS)' != 'Windows_NT'" />
```

Comment:
@mellinoe told me that these two were failing for different reasons in non-Windows builds but they in theory should work, he said that @chcosta was eventually going to look into those, so if it is getting removed already, I wonder if I should change the Configurations.props in order to get them filtered out or if we should keep this exclusion.